### PR TITLE
fix: lets load quickstarts from the version stream

### LINKED ISSUE #5283: support loading the quickstarts from the version stream
a common cause of confusion for folks using bitbucket / gitlab is that we currently use the github client to query the available quickstarts which means we need a github.com client + token.

It would simplify things greatly if we just loaded them from the version stream instead. That would also mean it would be easier to keep the quickstarts + their versions in sync in the version stream to improve quality

### PR DESCRIPTION
**Pull request by jstrachan**
fixes #5283

### CODE REVIEW

File: pkg/cmd/step/create/pr/step_create_pr_quickstarts.go [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+	"github.com/jenkins-x/jx/pkg/cmd/helper"
+
+	"github.com/jenkins-x/jx/pkg/cmd/opts"
+	"github.com/jenkins-x/jx/pkg/cmd/templates"
+	"github.com/spf13/cobra"
+)
+
+var (
+	createPullRequestQuickStartsLong = templates.LongDesc(`
+		Creates a Pull Request on a 'jx boot' git repository to mirror all the SourceRepository CRDs into the quickStarts Chart
```
Comment (pmuir) [inline pkg/cmd/step/create/pr/step_create_pr_quickstarts.go]: copy and paste ;-)

**Comment by jstrachan** [comment]
@pmuir think I've fixed those issues now thanks

**Pull request merged**